src/utils.ts

```diff
@@ -11,10 +11,21 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+import * as path from 'path';
 import {parse as parseUrl_, Url} from 'url';
 
 const unspecifiedProtocol = '-:';
 
+export const EOL = /\r\n|\r|\n/g;
```

Comment:
Okay I'm *old*.  `\r` by itself ended at Mac OS 9.  And nobody writes on Commodores now so I have removed `\r`.